Library/Formula/folly.rb

```diff
@@ -0,0 +1,39 @@
+class Folly < Formula
+  homepage "https://github.com/facebook/folly"
+  url "https://github.com/facebook/folly/archive/v0.22.0.tar.gz"
+  sha256 "de365f191cd88eada031c30372ff1bdf8bfeed7e3e8e4408b4695cb12ca5ab05"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
+  depends_on "double-conversion"
+  depends_on "glog"
+  depends_on "gflags"
+  depends_on "boost"
+  depends_on "libevent"
+
+  def install
+    system "autoreconf", "-i"
```

Comment:
Oops. I accidentally deleted this line when cleaning out the formula before a commit.